.github/workflow_scripts/env_setup.sh

```diff
@@ -58,7 +58,7 @@ function install_multimodal {
     # launch different process for each test to make sure memory is released
     python3 -m pip install --upgrade pytest-xdist
     python3 -m pip install --upgrade -e multimodal/[tests]
-    mim install mmcv-full --default-timeout 120
```

Comment:
@FANGAreNotGnu added this flag in this [PR](https://github.com/awslabs/autogluon/pull/2196) as suggested by @bryanyzhu . I guess it is to handle the timeout issue when installing mmcv. Is there a reason that we remove it?

Reply:
It is causing CI to fail.

Reply:
May be @Raldir we can try --timeout 1000.

Reply:
120 can still be too small I think. May be we can also revise it in the next PR.

Reply:
Ok, will revise to 1000